### DISCUSSION TITLE
Compose?

### DISCUSSION THREAD

**Discussion by trindadedev13** [aashishksahu/SafeSpace#99] [General]
I saw that the application has the compose libs, where are you using compose?

**Comment by aashishksahu** [comment]
It's everywhere, the app is built on jetpack compose.

**Reply by trindadedev13** [reply]
why settings is xml then

**Reply by aashishksahu** [reply]
So, earlier the app was entirely xml view based, then I migrated to Jetpack Compose. Some parts are still in xml because migrating them didn’t seem necessary.

**Reply by trindadedev13** [reply]
oh
